kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1551,9 +1551,40 @@ components:
           type: string
         defaultValue:
           type: string
+        source:
+          $ref: "#/components/schemas/Source"
+        isSensitive:
+          type: boolean
+        isReadOnly:
+          type: boolean
+        synonyms:
+          type: array
+          items:
+            $ref: "#/components/schemas/ConfigSynonym"
       required:
         - name
 
+    ConfigSynonym:
+      type: object
+      properties:
+        name:
+          type: string
+        value:
+          type: string
+        source:
+          $ref: "#/components/schemas/Source"
+
+    Source:
+      type: string
+      enum:
+        - DYNAMIC_TOPIC_CONFIG
+        - DYNAMIC_BROKER_LOGGER_CONFIG
+        - DYNAMIC_BROKER_CONFIG
+        - DYNAMIC_DEFAULT_BROKER_CONFIG
+        - STATIC_BROKER_CONFIG
+        - DEFAULT_CONFIG
+        - UNKNOWN
```

Comment:
Please merge master to your branch, I already added such classes there :)

Reply:
Done, I had to resolve some conflicts, but I'm now using your classes